esi_leap/api/controllers/v1/utils.py

```diff
@@ -0,0 +1,84 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import oslo_policy
+
+from esi_leap.common import exception
+from esi_leap.common import policy
+
+
+def contract_get_all_authorize_filters(cdict, r_project_id,
+                                       start_time=None, end_time=None,
+                                       status=None, offer_uuid=None,
+                                       project_id=None, view=None, owner=None):
```

Comment:
I think the bulk of this method should rename in contract.py; however there are definitely some bits that can be broken out into util functions (mentioned below).